docs/docs/scheme-intro/input-output.md

```diff
@@ -159,7 +159,7 @@ If you first evaluated the code that writes into a file `test-2.scm` you should
 was written into it. Note that `read` was called two times and read two expressions (two strings).
 
 Same as with output-port you need to call `close-input-port` procedure to close the input port. But
-Scheme, same as with output port have handy function that allow to close the port automagically.
+Scheme, same as with output port have handy function that allow to close the port automatically.
```

Comment:
You didn't fixed this:

This is not an error, see ["automagically" in Jargon File](http://www.catb.org/jargon/html/A/automagically.html).